mkosi/config.py

```diff
@@ -766,6 +766,14 @@ def config_default_dependencies(namespace: argparse.Namespace) -> Optional[list[
     return dependencies
 
 
+def config_default_image_version(namespace: argparse.Namespace) -> Optional[str]:
+    if not namespace.image_version_file:
+        return None
+
+    version_file = cast(Path, namespace.image_version_file)
+    return version_file.read_text().rstrip("\n")
```

Comment:
I think it makes more sense to just strip all whitespace to the right
```suggestion
    return version_file.read_text().rstrip()
```
Though it would probably be better to strip it all
```suggestion
    return version_file.read_text().strip()
```